🐛 [Bug] Plug in test failed

##  Bug Description
```
FAILED test_cuda_kernel_op.py::test_custom_geometry_aot_path_only - cuda.core._utils.cuda_utils.CUDAError: CUDA_ERROR_UNSUPPORTED_PTX_VERSION: This indicates that the provided PTX was compiled with an unsupported toolchain.
FAILED test_cuda_kernel_op.py::TestMultiOutput::test_schema_returns_tuple - assert False
 +  where False = any(<generator object TestMultiOutput.test_schema_returns_tuple.<locals>.<genexpr> at 0x7f4c206ec3c0>)
FAILED test_cuda_kernel_op.py::TestMultiOutput::test_eager_unpacks_to_two_tensors - AttributeError: '_OpNamespace' 'ttk_kp' object has no attribute 'sin_cos'
FAILED test_cuda_kernel_op.py::TestMultiOutput::test_trt_compile - AttributeError: '_OpNamespace' 'ttk_kp' object has no attribute 'sin_cos'
================= 23 failed, 25 passed, 177 warnings in 20.32s =================

```
https://github.com/pytorch/TensorRT/actions/runs/27276832297/job/80605163399

## To Reproduce

Steps to reproduce the behavior:

1.
2.
3.

<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0):
 - PyTorch Version (e.g. 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->

--- a/py/torch_tensorrt/kernels/_derive.py
+++ b/py/torch_tensorrt/kernels/_derive.py
@@ -425,10 +425,20 @@ def _compile_kernel(spec: KernelSpec) -> Tuple[bytes, Any, Any]:
     options = ProgramOptions(
         std=spec.compile_std, arch=arch, include_path=include_paths
     )
-    program = Program(spec.kernel_source, code_type="c++", options=options)
-    module = program.compile("ptx", name_expressions=(spec.kernel_name,))
-    ptx: bytes = module.code
-    kernel = module.get_kernel(spec.kernel_name)
+    # Two Program instances because cuda.core consumes name_expressions on the
+    # first compile() and raises NVRTC_ERROR_NO_NAME_EXPRESSIONS_AFTER_COMPILATION
+    # on the second call.
+    ptx_program = Program(spec.kernel_source, code_type="c++", options=options)
+    ptx: bytes = ptx_program.compile("ptx", name_expressions=(spec.kernel_name,)).code
+    # Use CUBIN — not PTX — to materialize the loadable kernel for the eager
+    # path.  Loading PTX goes through the driver's PTX JIT, which raises
+    # CUDA_ERROR_UNSUPPORTED_PTX_VERSION when the host driver predates the PTX
+    # ISA emitted by NVRTC (e.g. CI hosts on an older CUDA driver running
+    # cuda-python wheels built against a newer CUDA toolkit). CUBIN is loaded
+    # as-is — no driver-side PTX compatibility check.
+    cubin_program = Program(spec.kernel_source, code_type="c++", options=options)
+    cubin_module = cubin_program.compile("cubin", name_expressions=(spec.kernel_name,))
+    kernel = cubin_module.get_kernel(spec.kernel_name)
     return ptx, device, kernel
 
 

--- a/py/torch_tensorrt/kernels/_nvrtc.py
+++ b/py/torch_tensorrt/kernels/_nvrtc.py
@@ -41,7 +41,10 @@ def compile_to_ptx(
     """Compile CUDA C++ source to PTX using NVRTC via cuda-python.
 
     Returns:
-        (ptx_bytes, device, kernel_object)
+        (ptx_bytes, device, None) — the third slot is reserved for a loadable
+        kernel handle but is intentionally not materialized here; see
+        ``_derive._compile_kernel`` if you need one (it compiles to CUBIN to
+        avoid driver-side PTX JIT).
     """
     Device, Program, ProgramOptions, _launch, _LaunchConfig = _cuda_core_imports()
 
@@ -57,8 +60,10 @@ def compile_to_ptx(
     program = Program(kernel_source, code_type="c++", options=options)
     module = program.compile("ptx", name_expressions=(kernel_name,))
     ptx: bytes = module.code
-    kernel = module.get_kernel(kernel_name)
     _LOGGER.debug(
         "Compiled kernel '%s' to PTX for %s (%d bytes)", kernel_name, arch, len(ptx)
     )
-    return ptx, device, kernel
+    # Materializing a Kernel from the PTX module triggers the driver's PTX JIT,
+    # which fails with CUDA_ERROR_UNSUPPORTED_PTX_VERSION when the host driver
+    # is older than the PTX ISA NVRTC emits. Callers only consume ``ptx``.
+    return ptx, device, None

